moa-quickstart.md

```diff
@@ -0,0 +1,338 @@
+. /etc/bash_completion.d/moactl
+```
+
+Verify your installation by running the following command:
+
+```
+$ moactl
+Command line tool for MOA.
+
+Usage:
+  moactl [command]
+
+Available Commands:
+  completion  Generates bash completion scripts
+  create      Create a resource from stdin
+  delete      Delete a specific resource
+  describe    Show details of a specific resource
+  edit        Edit a specific resource
+  help        Help about any command
+  init        Applies templates to support Managed OpenShift on AWS clusters
+  list        List all resources of a specific type
+  login       Log in to your Red Hat account
+  logout      Log out
+  logs        Show logs of a specific resource
+  verify      Verify resources are configured correctly for cluster install
+  version     Prints the version of the tool
+
+Flags:
+      --debug     Enable debug mode.
+  -h, --help      help for moactl
+  -v, --v Level   log level for V logs
+
+Use "moactl [command] --help" for more information about a command.
+```
+
+Run the following command to verify that your AWS account has the necessary permissions:
+
+```
+$ moactl verify permissions
+I: Validating SCP policies...
+I: AWS SCP policies ok
+```
+
+If you encounter errors, check the [required AWS permissions](required-aws-permissions.md) documentation for more details.
+
+Verify that your AWS account has necessary quota to deploy an OpenShift cluster. Sometimes quota varies by region, which may prevent you from deploying:
+
+```
+$ export AWS_DEFAULT_REGION=us-west-2 && moactl verify quota
+I: Validating AWS quota...
+E: Insufficient AWS quotas
+E: Service ebs quota code L-FD252861 Provisioned IOPS SSD (io1) volume storage not valid
+```
+
+If needed, try another region:
+```
+$ export AWS_DEFAULT_REGION=us-east-2 && moactl verify quota
+I: Validating AWS quota...
+I: AWS quota ok
+```
+
+Once both the permissions and quota checks pass, proceed to initializing your AWS account.
+
+### Initializing your AWS account
+
+This step runs a CloudFormation template that prepares your AWS account for OpenShift deployment and management. This step typically takes 1-2 minutes to complete.
+
+*Link to CCS docs on not touching this?*
+
+```
+$ moactl init
+I: Logged in as 'rh-sd-jeder' on 'https://api.openshift.com'
+I: Validating AWS credentials...
+I: AWS credentials are valid!
+I: Validating SCP policies...
+I: AWS SCP policies ok
+I: Validating AWS quota...
+I: AWS quota ok
+I: Ensuring cluster administrator user 'osdCcsAdmin'...
+I: Admin user 'osdCcsAdmin' created successfuly!
+I: Verifying whether OpenShift command-line tool is available...
+E: OpenShift command-line tool is not installed.
+Go to https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/ to download the OpenShift client and add it to your PATH.
+```
+
+## Create your cluster
+
+Run the following command to create your cluster:
+
+```
+$ time moactl create cluster --name=rh-moa-test1
```

Comment:
Maybe we want to print out the output of `moactl create cluster --help` here, as it gives a bunch of options for the cluster

Reply:
I'm 50/50 on this. I think for the hackday audience it might be nice to offer a quick command they can copy/paste, but call out that they can run moactl create cluster --help to see all of their available options.